package.json

```diff
@@ -17,7 +17,8 @@
     "test:flow": "flow",
     "test:unit": "jest",
     "test:coveralls": "jest --coverage && cat ./coverage/lcov.info | coveralls",
-    "test": "npm run test:lint && npm run test:coveralls && npm run test:flow"
+    "test": "npm run test:lint && npm run test:coveralls && npm run test:flow",
+    "test-int": "mocha --compilers js:babel-core/register __int-tests__/basic.js"    
```

Comment:
Also I'd probably rename 'test-int' to something a bit more descriptive like 'test-appium'